README.md

```diff
@@ -1,27 +1,42 @@
-# WebAssembly Community Group: Design
+# WebAssembly Design
 
-This repository contains documents describing the high-level design of
-WebAssembly and is part of the 
-[WebAssembly Community Group](https://www.w3.org/community/webassembly/).
+> This repository contains documents describing the design and high-level overview of WebAssembly.
 
-The actual WebAssembly specification is being developed in the
-[spec repository](https://github.com/WebAssembly/spec/). High-level design
-discussions should continue to be held in the design repository for now so
-that the spec work can remain focused.
+> The documents and discussions in this repository are part of the [WebAssembly Community Group](https://www.w3.org/community/webassembly/).
 
-**Expect the contents to be in flux: everything is still tentative.**
+## Overview
 
-A good starting point is the [high-level design goals](HighLevelGoals.md),
-followed by the [FAQ](FAQ.md).
+WebAssembly or wasm is a portable, low-level, binary-format abstract syntax tree designed as an efficient target language for compilers.
```

Comment:
@jfbastien I would like the project to articulate clearly if it is developing a source code and if so to keep the language used consistent with this? I understand the need for a binary encoding, but this could be understood as a reversible encoding of the source code. I personally don't want to see a web machine code because I don't think this would be a good outcome for web users rights and I suspect a lot of the web community would agree, but I am fine putting this to a vote.

The decision making process is not great from my perspective. I have been fighting for over a year now on the performance path. I was in the process of trying to improve the type derivation of JS to better optimize access to the linear memory (typed array), and was making contributions across Odin/Ion/V8/JSC to try and get them all up to speed. You can see these discussions and conflicts of opinion still continuing in the issue I have raised in this group - there has been little progress over the last year. A binary encoding does not solve all these issues, and I made this point over a year ago, they are largely orthogonal matters. Many of the teams must have known my significant interest in the direction but the first I knew about wasm was the public announcement. The discussions on some of the issues raised in this group seem to me to come down to a small group claiming they know better, even when the performance issues have been demonstrated technically. I believe you yourself have closed issues I opened on the basis that is looked like a wall of words. I am also concerned that people are withholding ideas which makes discussions and progress difficult and complicating the cooperation in the group.

This group has not articulated a decision making process which I believe would be the fair and professional thing to do so that people know where they stand and thus if we want to participate and support this project.

I am not claiming I have all the answers or that my technical ideas would pan out but I doubt anyone here can claim to have all the answers either. People make mistakes, others in the group can help on the technical matters - it's a process and the groups documents can integrated these.

The subjective matters are even more problematic, but I personally would like to see this project develop a source code (with a binary encoding), and to stay away from any claims that it is a machine code which seems just too loaded in the practical environment of web, and I don't think this would practically impact the use case of being a compiler target.